webui/src/lib/api/index.js

```diff
@@ -144,7 +140,7 @@ class Auth {
         const response = await fetch(`${API_ENDPOINT}/auth/password`, {
             headers: new Headers(defaultAPIHeaders),
             method: 'POST',
-            body: json({token: token, newPassword: newPassword, email: email})
+            body: JSON.stringify({token: token, newPassword: newPassword, email: email})
```

Comment:
Is `JSON.stringify(data, null, "")` equal to `JSON.stringify(data)` ?

Reply:
Yea, same.